control-plane-pki-operator/certificaterevocationcontroller/certificaterevocationcontroller.go

```diff
@@ -197,6 +197,8 @@ func signerClassForSecretName(name string) (certificates.SignerClass, bool) {
 	switch name {
 	case manifests.CustomerSystemAdminSigner("").Name:
 		return certificates.CustomerBreakGlassSigner, true
+	case manifests.SRESystemAdminSigner("").Name:
```

Comment:
We could potentially change the `library-go` implementation to divulge the relationship between these resources with labels instead of doing this mapping, which would simplify things and make it so that the rotation controller configuration is the sole source of truth.